examples/secure_cloud_run/variables.tf

```diff
@@ -71,10 +71,10 @@ variable "organization_id" {
 variable "resource_names_suffix" {
   description = "A suffix to concat in the end of the network resources names."
   type        = string
-  default     = null
```

Comment:
Can't be null because it will throw an error:
"Invalid template interpolation value. Var.resource_names_suffix is null"